recipes/prettier/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "prettier" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/prettier/prettier/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 31c22f314066d3997aeafb345ebc1143179500d62f2334efc965ccdc2b4ffcf7
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: npm install -g $(npm pack)
+
+requirements:
+  host:
+    - nodejs 12  # [not (osx and arm64)]
+    - nodejs 15  # [osx and arm64]
```

Comment:
Also have a look at `vsce`.

Reply:
Was this comment related to this particular line or the the entire `requirements` block?

Specifying the `nodejs` version seems necessary as I am receiving the following error (on `osx`):

```
failed to get install actions, retrying.  exception was: Unsatisfiable dependencies for platform osx-64: {"nodejs[version='12.*|>=12.19.0,<13.0a0']"}
```

Reply:
Can you paste the output of `conda mambabuild` for this error? The version spec is satisfiable as listed thus there is probably a different problem.